currencies.php

```diff
@@ -0,0 +1,574 @@
+<?php
+
+class CampTix_Currency {
+
+	public $version = 20180627;
+
+	/**
+	 * @return array List of currencies with their labels. We are keeping combination
+	 * of all currencies supported by union of payment gateways to have a single source of truth.
+	 */
+	private static function get_currency_list() {
+		return array(
+			'AED' => array(
+				'label'  => __( 'United Arab Emirates dirham', 'camptix' ),
```

Comment:
For some of these currencies, part of the name is in lower case. Others have title case (each first letter capitalized). We should be consistent. I think title case would be better.